doc/development_process/project_roles.rst

```diff
@@ -49,6 +48,26 @@ Contributors are granted the following rights and responsibilities:
 * Responsibility to follow the project’s code of conduct
   (https://github.com/zephyrproject-rtos/zephyr/blob/main/CODE_OF_CONDUCT.md)
 
+Contributors are initially only given `Read
+<https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories/repository-permission-levels-for-an-organization>`_
+access to the Zephyr GitHub repository. Specifically, at the Read access level,
+Contributors are not allowed to assign reviewers to their own pull-requests. An
```

Comment:
I think we generally us "pull request" without a hyphen. (Applies elsewhere in the PR.)

Reply:
There was existing uses of "pull-request" in this file, but "pull request" (no hyphen) seems to be more widely used so I fixed all references in this file.